const e=JSON.parse('{"key":"v-7f4a0796","path":"/exception/gailan.html","title":"一文彻底搞懂Java异常处理，YYDS","lang":"zh-CN","frontmatter":{"title":"一文彻底搞懂Java异常处理，YYDS","shortTitle":"Java异常处理全面解析","category":["Java核心"],"tag":["异常处理"],"description":"本文详细介绍了Java异常处理的基本概念、种类和原则，以及如何使用try-catch-finally和throw、throws关键字进行异常处理。通过实际案例和代码示例，帮助读者掌握Java异常处理的关键技巧，提高编程中的错误检测与处理能力。","head":[["meta",{"name":"keywords","content":"Java,Java异常处理, try-catch-finally, throw, throws,异常处理,java 异常处理"}],["meta",{"property":"og:url","content":"https://javabetter.cn/exception/gailan.html"}],["meta",{"property":"og:site_name","content":"二哥的Java进阶之路"}],["meta",{"property":"og:title","content":"一文彻底搞懂Java异常处理，YYDS"}],["meta",{"property":"og:description","content":"本文详细介绍了Java异常处理的基本概念、种类和原则，以及如何使用try-catch-finally和throw、throws关键字进行异常处理。通过实际案例和代码示例，帮助读者掌握Java异常处理的关键技巧，提高编程中的错误检测与处理能力。"}],["meta",{"property":"og:type","content":"article"}],["meta",{"property":"og:locale","content":"zh-CN"}],["meta",{"property":"og:updated_time","content":"2023-12-08T12:03:00.000Z"}],["meta",{"property":"article:author","content":"沉默王二"}],["meta",{"property":"article:tag","content":"异常处理"}],["meta",{"property":"article:modified_time","content":"2023-12-08T12:03:00.000Z"}],["script",{"type":"application/ld+json"},"{\\"@context\\":\\"https://schema.org\\",\\"@type\\":\\"Article\\",\\"headline\\":\\"一文彻底搞懂Java异常处理，YYDS\\",\\"image\\":[\\"\\"],\\"dateModified\\":\\"2023-12-08T12:03:00.000Z\\",\\"author\\":[{\\"@type\\":\\"Person\\",\\"name\\":\\"沉默王二\\",\\"url\\":\\"/about-the-author/\\"}]}"]]},"headers":[{"level":3,"title":"01、什么是异常","slug":"_01、什么是异常","link":"#_01、什么是异常","children":[]},{"level":3,"title":"02、Exception和Error的区别","slug":"_02、exception和error的区别","link":"#_02、exception和error的区别","children":[]},{"level":3,"title":"03、checked和unchecked异常","slug":"_03、checked和unchecked异常","link":"#_03、checked和unchecked异常","children":[]},{"level":3,"title":"04、关于 throw 和 throws","slug":"_04、关于-throw-和-throws","link":"#_04、关于-throw-和-throws","children":[]},{"level":3,"title":"05、关于 try-catch-finally","slug":"_05、关于-try-catch-finally","link":"#_05、关于-try-catch-finally","children":[]},{"level":3,"title":"06、小结","slug":"_06、小结","link":"#_06、小结","children":[]}],"git":{"createdTime":1626159673000,"updatedTime":1702036980000,"contributors":[{"name":"itwanger","email":"www.qing_gee@163.com","commits":15},{"name":"沉默王二","email":"www.qing_gee@163.com","commits":7},{"name":"shuimuju","email":"73344468+shuimuju@users.noreply.github.com","commits":1}]},"readingTime":{"minutes":16.62,"words":4985},"filePathRelative":"exception/gailan.md","localizedDate":"2021年7月13日"}');export{e as data};
